block/src/Pos/Arbitrary/Block.hs

```diff
@@ -153,7 +152,7 @@ instance (HasConfiguration, Arbitrary SscPayloadDependsOnSlot) =>
         return $ T.MainBody txPayload mpcData dlgPayload mpcUpload
 
 instance (HasConfiguration, Arbitrary SscPayload) => Arbitrary (T.Body T.MainBlockchain) where
-    arbitrary = makeSmall genericArbitrary
```

Comment:
Doesn't it make tests significantly slower?

Reply:
Sure, having a bigger thing could lead to slower tests. Here I'm benchmarking though; I'm very interested in whether things are significantly slower in relation to size. Leaving in `makeSmall` made it very hard for me to get useful data.